tests/ignite/contrib/metrics/test_cohen_kappa.py

```diff
@@ -173,18 +173,17 @@ def get_test_cases():
 def _test_distrib_binary_input(device):
 
     rank = idist.get_rank()
-    torch.manual_seed(12)
+    n_iters = 80
```

Comment:
Same remark here as for your previous PR : https://github.com/pytorch/ignite/pull/2655#pullrequestreview-1078345358